packages/velaux-ui/src/pages/ApplicationInstanceList/components/ContainerLog/index.tsx

```diff
@@ -126,7 +142,18 @@ class ContainerLog extends Component<Props, State> {
         onClose={this.props.onClose}
         overflowScroll
         v2
-        title={<Translation>Container Log</Translation>}
+        title={
+          <Row style={{ width: '100%' }}>
+            <Col span={12}>
+              <Translation>Container Log</Translation>
+            </Col>
+            <Col span={12}>
+            <Button style={{float: "right"}} type="normal" size="small" onClick={this.downloadLog}>
+              <Translation className="font-bold font-size-14">download</Translation>
```

Comment:
Let's use the AiOutlineDownload icon to replace the text.

Reply:
Done, and the preview is updated